puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/test/testKatello.groovy

```diff
@@ -0,0 +1,170 @@
+
+                    sh "cp $HOME/postgresql.db.yaml config/database.yml"
+
+                    sh "sed -i \"s/database:.*/database: ${gemset()}-test/\" config/database.yml"
+                    sh """
+cat <<EOT >> config/database.yml
+development:
+  adapter: postgresql
+  database: ${gemset()}-development
+  username: foreman
+  password: foreman
+  host: localhost
+  template: template0
+
+production:
+  adapter: postgresql
+  database: ${gemset()}-development
+  username: foreman
+  password: foreman
+  host: localhost
+  template: template0
+EOT
+                    """
+                }
+
+            }
+        }
+        stage('Configure Database') {
+            steps {
+
+                dir('foreman') {
+
+                    withRVM(['bundle update'], ruby)
+                    withRVM(['bundle exec rake db:create -q'], ruby)
+                    withRVM(['bundle exec rake db:migrate -q'], ruby)
+
+                }
+
+            }
+        }
+        stage('Run Tests') {
+            steps {
+
+                parallel (
+                    'tests': { dir('foreman') { withRVM(['bundle exec rake jenkins:katello TESTOPTS="-v"'], ruby) } },
+                    'rubocop': { dir('foreman') { withRVM(['bundle exec rake katello:rubocop TESTOPTS="-v" --trace'], ruby) } },
+                    'react-ui': {
+                        script {
+                            if (fileExists('package.json')) {
+                                sh 'npm install'
+                                sh 'npm test'
+                            }
+                        }
+                    },
+                    'angular-ui': {
+                        dir('engines/bastion_katello') {
+
+                            withRVM(['bundle show bastion > bastion-version'], ruby)
+
+                            script {
+                                bastion_install = readFile('bastion-version')
+                                bastion_version = bastion_install.split('bastion-')[1]
+                            }
+
+                            sh "cp -rf \$(cat bastion-version) bastion-${bastion_version}"
+                            sh "npm install npm"
+                            sh "node_modules/.bin/npm install bastion-${bastion_version}"
+                            sh "grunt ci"
+                        }
+                    },
+                    'assets-precompile': { dir('foreman') {
+                            withRVM(['bundle exec rake plugin:assets:precompile[katello] RAILS_ENV=production --trace'], ruby)
+                        }
+                    },
+                )
+
+            }
+        }
+        stage('Test db:seed') {
+            steps {
+
+                dir('foreman') {
+
+                    withRVM(['bundle exec rake db:drop db:create'], ruby)
+                    withRVM(['bundle exec rake db:migrate'], ruby)
+                    withRMV(['bundle exec rake db:seed'], ruby)
```

Comment:
@ehelms if you write this as `withRVM` instead of `withRMV`, the `db:seed` tests will also pass :laughing:

Reply:
I knew you'd know the reason! :)